gandi_dyndns.py

```diff
@@ -317,7 +317,8 @@ def update_ip():
       }, {
         'name': new_dynamic_record['name'],
         'type': new_dynamic_record['type'],
-        'value': external_ip
+        'value': external_ip,
+	'ttl': 600
```

Comment:
This should be 5 minutes as discussed above, and the indentation needs to match the surrounding lines.